packages/styleguide/Blocks/StoryStatus.tsx

```diff
@@ -2,54 +2,28 @@ import React from 'react';
 import { styled } from '@storybook/theming';
 import { colors } from '@codecademy/gamut-styles/utils/variables';
 
-const Statuses = {
-  ready: { message: 'Production Ready', color: colors.green[500] },
-  inProgress: { message: 'Use with Caution (WIP)', color: colors.yellow[500] },
-  deprecated: { message: 'Do not use (Deprecated)', color: colors.red[500] },
+const STATUS_COLORS = {
+  stable: colors.green[500],
+  volatile: colors.yellow[500],
+  deprecated: colors.red[500],
```

Comment:
These are a little bright for backgrounds behind white text. Can we darken them a bit?

Accessibility issue, for me 😄 

```suggestion
  stable: colors.green[800],
  volatile: colors.yellow[700],
  deprecated: colors.red[700],
```